src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -8,6 +8,10 @@ import modelperformancemonitoringOpenAIDashboard1 from 'images/model-performance
 
 With our OpenAI integration, you have the ability to seamlessly monitor all OpenAI queries. By adding just two lines of code, your team can gain access to key performance metrics such as cost, response time, rate limits, error rates, chat history, and sample inputs and outputs. Our OpenAI integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
 
+<Callout variant="important">
+This installation method has been replaced by our new AI monitoring integration. We recommend using that integration instead for new installations. [See our docs to get started](/docs/ai-monitoring/intro-to-ai-monitoring/).
```

Comment:
I would rewrite the first sentence to avoid the passive voice: `Our new AI monitoring integration replaces this installation method.`

Reply:
Good call re:passive. Ended up going with:

> To instrument your OpenAI-powered app, we recommend using AI monitoring rather than this older installation method. [See our docs to get started](/docs/ai-monitoring/intro-to-ai-monitoring/).